src/System.IO.Ports/tests/Legacy/SerialPort/ReadTimeout.cs

```diff
@@ -36,146 +36,167 @@ private enum ThrowAt { Set, Open };
 
     #region Test Cases
 
+    [ActiveIssue(15961)]
     [ConditionalFact(nameof(HasLoopbackOrNullModem))]
     public void ReadTimeout_Default_Read_byte_int_int()
```

Comment:
I don't think you need to disable anything which is already marked HasLoopbackOrNullModem, because they won't be running on CI machines anyway.